libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/events/NavigationFeedbackEvent.kt

```diff
@@ -20,11 +20,22 @@ internal class NavigationFeedbackEvent(
     var feedbackType: String? = null
     var source: String? = null
     var description: String? = null
-    var locationsBefore: Array<Location>? = emptyArray()
-    var locationsAfter: Array<Location>? = emptyArray()
+    var locationsBefore: Array<FeedbackLocation>? = emptyArray()
+    var locationsAfter: Array<FeedbackLocation>? = emptyArray()
     var screenshot: String? = null
     var feedbackSubType: Array<String>? = emptyArray()
     var appMetadata: AppMetadata? = null
 
     override fun getEventName(): String = NavigationMetrics.FEEDBACK
 }
+
+internal data class FeedbackLocation(
+    @SerializedName("lat") val latitude: Double,
+    @SerializedName("lng") val longitude: Double,
+    @SerializedName("speed") val speed: Float,
+    @SerializedName("course") val course: Float,
+    @SerializedName("altitude") val altitude: Double,
+    @SerializedName("timestamp") val timestamp: Long,
+    @SerializedName("horizontalAccuracy") val horizontalAccuracy: Float,
+    @SerializedName("verticalAccuracy") val verticalAccuracy: Float
```

Comment:
This should be nullable if not present.

Reply:
@LukasPaczos I set it `0.0` if not present,  because `horizontalAccuracy ` has the next docs:
`If this location does not have a horizontal accuracy, then 0.0 is returned.`
so, I just implemented the same for `verticalAccuracy `.

do you prefer `null` value instead?

Reply:
I rechecked the schema and it requires a number, it cannot be null, so I think passing 0 is fine :+1: